tests/test_pump_scheduler.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+import unittest
+import datetime
+
+import mock
+
+from greenpithumb import pump_scheduler
+
+
+class PumpSchedulerTest(unittest.TestCase):
+
+    def setUp(self):
+        self.mock_clock = mock.Mock()
+
+    def test_wake_hour_less_than_sleep_hour(self):
+        """Test rollover from one day to the next.
+        
+        If the sleep period spans two days, the pump should not run between 
+        midnight and the wake hour.
+        """
+        self.mock_clock.now.return_value = datetime.datetime(2016, 5, 24, 0)
+        sleep_windows = ((22, 8),)
```

Comment:
(minor, optional) This is correct, but looks kind of weird. Suggest changing it to a list (square brackets) just because a list is fine here too and the syntax looks a little clearer like:

```python
sleep_windows = ([22,8])
```

(ditto for the rest, just for consistency)